File: Lat2/Python/Memory.py
from Hardware import Hardware
class Memory(Hardware):
    def __init__(self):
        self.cuda = ""
        self.size = ""
        self.frequency = ""
    def setCuda(self, cuda):
        self.cuda = cuda
    def getCuda(self):
        return self.cuda
    def setSize(self, size):
        self.size = size
    def getSize(self):
        return self.size
    def setFrequency(self, frequency):
        self.frequency = frequency
    def getFrequency(self):
        return self.frequency

File: Lat2/PHP/Memory.php
<?php
class Memory extends Hardware{
    private $cuda;
    private $size;
    private $frequency;
    function __construct()
    {
        
    }
    function setCuda($cuda){
        $this->cuda = $cuda;
    }
    function getCuda(){
        return $this->cuda;
    }
    function setSize($size){
        $this->size = $size;
    }
    function getSize(){
        return $this->size;
    }
    function setFrequency($frequency){
        $this->frequency = $frequency;
    }
    function getFrequency(){
        return $this->frequency;
    }
    function __destruct()
    {
        
    }
}
?>